bbj-vscode/src/language/bbj-scope.ts

```diff
@@ -200,7 +201,7 @@ export class BbjScopeProvider extends DefaultScopeProvider {
             // handle implicit extends java.lang.Object
             const javaObject = this.javaInterop.getResolvedClass('java.lang.Object');
             if (javaObject) {
-                return this.createCaseSensitiveScope(descriptions, this.createScopeForNodes(stream(javaObject.fields).concat(javaObject.methods)))
+                return this.createCaseSensitiveScope(descriptions, this.createScopeForNodes(stream(javaObject.fields as any[]).concat(javaObject.methods as any[])))
```

Comment:
`any` cast is not necessary. Also maybe it is better to extract the parameter to a local const. 
```suggestion
                const members = stream(javaObject.fields).concat(javaObject.methods)
                return this.createCaseSensitiveScope(descriptions, this.createScopeForNodes(members))
```